build.gradle

```diff
@@ -128,6 +128,9 @@ allprojects {
     assemble.dependsOn shadowJar
 }
 
+test.dependsOn generateUnstableTestResources
+check.dependsOn generateUnstableTestResources
```

Comment:
Is the second line needed? `test` is a dependency of `check`, so `generateUnstableTestResources` should transitively be a dependency also.